### ISSUE TITLE
Support within lists, admonitions, and content tabs

### ISSUE THREAD

**Issue by CollinHeist** [mkdocs-plugins/termynal#18]
Hello, love the project! Thanks for your hard work.

I think it would be great if the terminals could be embedded below lists, and within admonitions / content tabs.

Thanks!

**Comment by daxartio** [comment]
Hi!

I thought about it. I wanted to do interoperability with mkdocs-material.

**Comment by daxartio** [comment]
Have you wanted to see the result like this?

<img width="1366" alt="image" src="https://github.com/mkdocs-plugins/termynal/assets/11211587/7db10cc1-4f9f-4917-bdf0-9896991c6668">

**Comment by CollinHeist** [comment]
As far as raw HTML, yes that was what I was envisioning. After rendering, like this:

![image](https://github.com/mkdocs-plugins/termynal/assets/17693271/39313b98-577a-47e9-9e7e-d1116d9554bb)

After briefly looking at the code, I *think* the termynal regex would need to be modified to handle arbitrary indentation (that takes care of lists, admonitions, and tabs), as it currently only looks for left-aligned code blocks (no `\s*`)

**Comment by eshaben** [comment]
Yeah, I am running into the same problem. I have a numbered list that I would like to show the terminal output for each step of the way, but since the terminal doesn't render when indented, I can't use it 😔

**Comment by daxartio** [comment]
Since version 0.12, the plugin supports content tabs

**Comment by pbordron** [comment]
Works inside admonitions and lists except if you use [`pymdownx.arithmatex`](https://squidfunk.github.io/mkdocs-material/setup/extensions/python-markdown-extensions/?h=pymdownx+arithmatex#arithmatex) markdown extension with a termynal prompt containing a `$` (that also delimits math blocks in latex)

- with `pymdownx.arithmatex`

<img width="777" height="309" alt="Image" src="https://github.com/user-attachments/assets/3120a032-7ba7-4a5d-83df-8d17f7b939a4" />

- without `pymdownx.arithmatex`

<img width="777" height="317" alt="Image" src="https://github.com/user-attachments/assets/eeeab91b-f6d9-434a-b8e1-0835313660f9" />
